src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
+@style = @{ elements.args.get('_inLine) match { case Some(true) => "form-checkbox" case _ => "block-label" } }
+
+<label class="form-field-single @if(elements.args.get('_labelClass)){ @elements.args.get('_labelClass) } @style @if(elements.hasErrors){ error form-field--error } @if(value=="true" && style=="block-label"){ selected }" for="@elements.id">
     @elements.errors.map { error =>
-    <span style="display: block" class="error-notification">@Messages(error)</span>
+        <span id="@{elements.field.id}--error" class="error-notification" style="display: block">@Messages(error)</span>
     }
-    <input type="checkbox" id="@elements.id"
-    @if( elements.args.get('_inputClass) ){ class="@elements.args.get('_inputClass)" }
-    name="@elements.field.name" value="true" @if(value=="true"){checked="checked"}/>
-    <label class="checkbox@if(elements.args.get('_labelClass)){ @elements.args.get('_labelClass)}" for="@elements.id"> @elements.label</label>
-</div>
+    <input type="checkbox" id="@elements.id" name="@elements.field.name" value="true" @if(value=="true"){ checked="checked" } @if( elements.args.get('_inputClass) ){ class="@elements.args.get('_inputClass)" } @if(elements.args.contains('_dataAttributes)){ @elements.args.get('_dataAttributes) }>
+    @elements.label
```

Comment:
Placing the label before the input field would cause the checkboxes to look like that (in case a simple chexkbox is used):
![signedout3](https://cloud.githubusercontent.com/assets/8497852/22506163/d9bbc4b8-e876-11e6-8aa1-8c27b1224912.png)